quaddtype/numpy_quaddtype/__init__.py

```diff
@@ -40,3 +42,6 @@ def LongDoubleQuadPrecDType():
 epsilon = get_sleef_constant("epsilon")
 smallest_normal = get_sleef_constant("smallest_normal")
 smallest_subnormal = get_sleef_constant("smallest_subnormal")
+bits = SleefQuadPrecDType().itemsize * 8
+precision = int(-np.log10(epsilon))
```

Comment:
since `epsilon` is the sleef constant, you can use the `quad_log10` function instead of numpy and then just expose these in python and fetch them  here via `get_sleef_constant`.
That way if somewhere it'll be needed internally we can just utilize that avoiding recalculation.

Reply:
Maybe can do with other constants as well, and I think it'll be a good idea to keep all the constants inside the main source code right?